delphi-checks/src/test/java/au/com/integradev/delphi/checks/DigitGroupingCheckTest.java

```diff
@@ -62,6 +66,10 @@ void testNonStandardGroupingShouldAddIssue(String actual) {
         "%1_01_01",
         "%10_101",
         "%10_1010",
+        "&100_101",
```

Comment:
nit: these look like a binary literal at first, can you change the non-leading 1s to 0?